pkg/client/cache/listwatch_test.go

```diff
@@ -117,7 +117,7 @@ func TestListWatchesCanWatch(t *testing.T) {
 		{
 			location: buildLocation(
 				testapi.Default.ResourcePathWithPrefix("watch", "nodes", api.NamespaceAll, ""),
-				buildQueryValues(url.Values{"resourceVersion": []string{""}})),
+				buildQueryValues(url.Values{})),
```

Comment:
Just to explain this one - with VersionedParams "zero value" is not added to url.Values{} (and I think this is WAI).